tests/integration/phplabel.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+PHP label rule
+--FILE--
+<?php
+require 'vendor/autoload.php';
+
+use Respect\Validation\Validator as v;
+
+var_dump(v::phpLabel()->validate('person'));
+var_dump(v::phpLabel()->validate('4ccess'));
```

Comment:
This is already covered by the unit tests, would be nicer to test check and/or assert methods.